modules/mobile/app/models/mobile/v0/new_dependent_form_submission.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'common/models/resource'
+
+module Mobile
+  module V0
+    class NewDependentFormSubmission < Common::Resource
```

Comment:
The hardest thing is always naming things. I chose this name for the model and serializer because dependent was already taken and the data really isn't a dependent. It's just a form submission id. I think you can argue that it doesn't justify a model at all, but we've been trying to convert everything over to models in order to have attribute type validation. 

Let me know if you take issue with the naming convention or have better ideas. The other one I considered was `DependentCreationFormSubmission` but I thought that was slightly less clear.

Reply:
I think this naming is good